vcenter/vcenter.py

```diff
@@ -141,9 +150,38 @@ def undeploy(self, uuid):
         vm = self.content.searchIndex.FindByUuid(None, uuid, True)
         if vm:
             self.__logger.debug('found vm: {}'.format(vm.config.uuid))
+
+            parent_folder = vm.parent
+            self.__logger.debug('searching for sibling machines in: {}({})'.format(
+                                                                                    vm.parent,
+                                                                                    vm.parent.name
+            ))
+            objView = self.content.viewManager.CreateContainerView(
+                                                                    self.content.rootFolder,
+                                                                    [vim.VirtualMachine],
+                                                                    True
+            )
+
+            sibling_machines_count = 0
+            for item in objView.view:
```

Comment:
I think this can be written in more Pythonic way as:
```
sibling_machine_names = [str(item.name) for item in objView.view where item.parent == parent_folder and item.config.uuid != vm.config.uuid]
sibling_machines_count = len(sibling_machine_names )
self.__logger.debug('Sibling machines found in {}: {}'.format(parent_folder, ','.join(sibling_machine_names)))
```

Reply:
The pythonic way is just a syntactic sugar. Then just let's encapsulate it to method and it is done :]

Reply:
no problem with this

Reply:
fixed